test/medfilt.cpp

```diff
@@ -406,7 +408,8 @@ TEST(MedianFilter, Docs) {
     //    2.0000        6.0000       10.0000       14.0000
     //    3.0000        7.0000       11.0000       15.0000
     //    4.0000        8.0000       12.0000       16.0000
-    array b = medfilt(a, 3, 3, AF_PAD_ZERO);
+    array b;
+    try { b = medfilt(a, 3, 3, AF_PAD_ZERO); } catch FUNCTION_UNSUPPORTED
```

Comment:
These are examples in documentations. You should never put test related stuff in these types of code snippets.